src/threading/pss_common.h

```diff
@@ -1,112 +0,0 @@
-/*
```

Comment:
The Intel implementation that was introduced with the OpenMP back end caused no end of problems for some compilers. For example we couldn't use CUDA with the cthread back end.
Whatever parallel sort we use, it doesn't have to be the best of the best, just good enough to reduce model building time.